AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -638,7 +665,9 @@ extension MeetingModel: ActiveSpeakerObserver {
 
 extension MeetingModel: DataMessageObserver {
     func dataMessageDidReceived(dataMessage: DataMessage) {
-        chatModel.addDataMessage(dataMessage: dataMessage)
+        if dataMessage.topic == "chat" {
```

Comment:
what other topics do we support?

Reply:
Currently there're no other topics being used in the demo app. Previously added this expecting transcripts to come through data message as well but later figured it came through another channel. But I feel having a check here provides more clarity that "chat" is just one use case/topic of data message. There will be another "aws:chime:transcription" topic to handle transcripts once we migrate to a unified architecture.